packages/plugin-phone/src/web-rtc-media.js

```diff
@@ -12,13 +13,64 @@ import {
 } from './webrtc';
 
 /**
+ * Determines if the peer connection is receiving the specified kind of media
+ * @param {string} kind audio|video
+ * @param {RTCPeerConnection} pc
+ * @returns {bool} true if receiving, false if not
+ */
+function getRemoteMediaStatus(kind, pc) {
+  const streams = pc.getRemoteStreams();
+
+  if (streams.length === 0) {
+    return false;
+  }
+
+  const res = streams.reduce((areStreamsFlowing, stream) => {
```

Comment:
I hadn't used `some` before. I like it in principle, but my first attempt to switch broke all the tests, so i'm inclined to leave this as-is for now :)